sputnikdao2/src/types.rs

```diff
@@ -18,6 +18,7 @@ pub const GAS_FOR_FT_TRANSFER: Gas = Gas::from_tgas(10);
 /// Configuration of the DAO.
 #[derive(Clone, Debug, PartialEq)]
 #[near(serializers=[borsh, json])]
+#[serde(deny_unknown_fields)]
```

Comment:
Please, refactor into

```suggestion
#[near(serializers=[borsh, json(deny_unknown_fields)])]
```

Reply:
Sorry, this syntax is only supported for borsh, so I close this request